llvm/test/Transforms/InterleavedAccess/RISCV/interleaved-accesses.ll

```diff
@@ -44,11 +44,19 @@ define void @load_factor2_as(ptr addrspace(1) %ptr) {
 
 define void @load_factor2_vscale(ptr %ptr) {
 ; RV32-LABEL: @load_factor2_vscale(
-; RV32-NEXT:    [[TMP1:%.*]] = call { <vscale x 8 x i32>, <vscale x 8 x i32> } @llvm.riscv.vlseg2.nxv8i32.i32(<vscale x 8 x i32> poison, <vscale x 8 x i32> poison, ptr [[PTR:%.*]], i32 -1)
+; RV32-NEXT:    [[TMP1:%.*]] = call riscv_m4x2 @llvm.riscv.vlseg2.riscv_m4x2.i32(riscv_m4x2 poison, ptr [[PTR:%.*]], i32 -1, i32 5)
```

Comment:
These are not target extension types?

Reply:
This is actually the test case that updated in the old patch, I've removed it lol~